bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieRequestProcessor.java

```diff
@@ -705,4 +713,8 @@ public boolean isBlacklisted(Channel channel) {
     public void handleNonWritableChannel(Channel channel) {
         onResponseTimeout.accept(channel);
     }
+
+    private static void writeAndFlush(Channel channel, Object msg) {
```

Comment:
no need static.

Reply:
It's best practice to make a method static if possible. 
Look at this reference for example: https://stackoverflow.com/questions/2671496/when-to-use-static-methods

Reply:
I got, thanks.